dataactbroker/handlers/fileHandler.py

```diff
@@ -1170,7 +1170,8 @@ def submission_to_dict_for_status(submission):
         'reporting_period_start_date': reporting_date(submission),
         'reporting_period_end_date': reporting_date(submission),
         'jobs': [job_to_dict(job) for job in relevant_jobs],
-        'publish_status': submission.publish_status.name
+        'publish_status': submission.publish_status.name,
+        'quarterly_submission': submission.is_quarter_format
```

Comment:
Realized that things were added here but not updated in the docs here: https://github.com/fedspendingtransparency/data-act-broker-backend/tree/development/dataactbroker#post-v1check_status

We can use this PR to make sure the API docs are updated for the additions in the submission data that gets returned for `/check_status`